packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupSizeSpread.stories.tsx

```diff
@@ -12,162 +12,189 @@ const useStyles = makeStyles({
 
 export const SizeSpread = () => {
   const styles = useStyles();
-
   return (
     <div className={styles.root}>
       <AvatarGroup layout="spread" size={16}>
         <AvatarGroupItem name="Katri Athokas" />
         <AvatarGroupItem name="Elvia Atkins" />
         <AvatarGroupItem name="Cameron Evans" />
         <AvatarGroupItem name="Wanda Howard" />
-        <AvatarGroupItem name="Mona Kane" />
-        <AvatarGroupItem name="Allan Munger" />
-        <AvatarGroupItem name="Daisy Phillips" />
-        <AvatarGroupItem name="Robert Tolbert" />
-        <AvatarGroupItem name="Kevin Sturgis" />
+        <AvatarGroupOverflow>
+          <AvatarGroupItem name="Mona Kane" />
+          <AvatarGroupItem name="Allan Munger" />
+          <AvatarGroupItem name="Daisy Phillips" />
+          <AvatarGroupItem name="Robert Tolbert" />
+          <AvatarGroupItem name="Kevin Sturgis" />
+        </AvatarGroupOverflow>
       </AvatarGroup>
```

Comment:
Not new in this PR, but this is a lot of copy/pasted code 🥵. Maybe map an array of sizes instead?

```jsx
const sizes = [16, 20, ...];

return (
  <div className={styles.root}>
    {sizes.map(size => (
      <AvatarGroup layout="spread" size={size} key={size}>
        <AvatarGroupItem name="Katri Athokas" />
        ...
      </AvatarGroup>
    ))}
  </div>
);
```